### ISSUE TITLE
Livestreams should not have playback parameters (speed adjustment or skip slience)

### ISSUE THREAD

**Issue by litetex** [TeamNewPipe/NewPipe#7667]
### Checklist
- [x] I am using the latest version - 0.21.15
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug
1. Open a livestream

### Actual behavior
Notice that you can change the speed / skip silence / etc

### Expected behavior
None of the options should be available and a livestream, because it is LIVE.

### Screenshots/Screen recordings
![grafik](https://user-images.githubusercontent.com/40789489/149824814-a9b30d8f-42e7-4a07-8ad9-c320daa1c694.png)

**Comment by KosOrKosm** [comment]
Hello @litetex, can I take on this issue?

**Comment by TiA4f8R** [comment]
I understand why this feature should to be removed: it feel indeed useless to use them when playing a livestream close to the live edge.

But depending on the streams and/or platerforms, you may be able to rewind for a pretty good time, like some minutes or maybe even more (with TeamNewPipe/NewPipeExtractor#780, up to 1 hour for some streams on YouTube, thanks to the HLS manifest for the iOS client(s))!

So, I don't think that's a good idea at all. If we were not able to rewind on every livestream (that's the case for instance on some YouTube livestreams, only when creators disable this ability), I would be agree with this removal, but here I am not.

**Comment by litetex** [comment]
@TiA4f8R 

I'm pretty sure that a normal user will in 99% of all cases be annoyed if the livestream is not live and the stream e.g. starts buffering (>1x) or he gets the part where e.g. football/soccer team xy scores a hit 2mins too late (<1x).

If we don't hide the UI we should at least set the speed (at best temporarily for the live stream only) to 1x and let the UI still be visible.

Maybe we should also differ between "post" live streams and "live" live streams.

**Comment by KosOrKosm** [comment]
Perhaps the speed option can be hidden + reset to 1 when on the live edge, and shown otherwise? That way it can still be used after seeking the beginning of the stream or watching an archive, but also won't cause buffering on the live edge.

**Comment by TiA4f8R** [comment]
> Maybe we should also differ between "post" live streams and "live" live streams.

That's almost not possible for me, especially because you only know this when fetching the manifest/content.

> If we don't hide the UI we should at least set the speed (at best temporarily for the live stream only) to 1x and let the UI still be visible.

That's for me the thing we should go to, especially because ExoPlayer has special capabilities for our case (maybe not in the version we are using, but that's probably not an issue because we will update soon to one of the latest): https://exoplayer.dev/doc/reference/com/google/android/exoplayer2/MediaItem.LiveConfiguration.html

**Comment by opusforlife2** [comment]
So is there anything for @KosOrKosm to do right now, or should they wait until the Exo 2.17.1 PR is merged?

**Comment by MustkimKhatik** [comment]
Is there any assignee on this issue?
I want to get started with open source if there any other issue You can GUIDE me

**Comment by TiA4f8R** [comment]
Yes, there is one, see the PR linked. Take a look at other issues with the good first issue label.

**Comment by GameKyuubi** [comment]
> I'm pretty sure that a normal user will in 99% of all cases be annoyed if the livestream is not live and the stream e.g. starts buffering (>1x)

You shouldn't remove the speed function or skip silence for live, particularly if you're trying to catch up to get to the live segment. What you should do is automatically set it to 1x once you hit the live mark.

**Comment by dftf-stu** [comment]
> "What you should do is automatically set it to 1x once you hit the live mark"

Yes, this is how YouTube works: if you rewind a live video, it will play at 1.25x, 1.5x, 1.75x or 2.0x until you've caught-up to the point where the actual live bit of the broadcast is, then reverts to 1x.  NewPipe, however, instead waits until around 10-15 seconds of the video has gone-by, plays that part at your selected speed, then hangs again waiting to fill the buffer. 🙃

Though have to add: I rarely find any live videos in NewPipe that left me rewind... when they all do in the YouTube app or site.

**Comment by absurdlylongusername** [comment]
I think this issue should just be closed. If such features exist on youtube for livestreams, they should exist in NewPipe. We shouldn't have _less_ functionality than youtube
